### DISCUSSION TITLE
Change the values displayed in the interface scale slider for consistency

### DISCUSSION THREAD

**Discussion by Calinou** [redeclipse/base#1186] [Ideas and Feature Requests]
While browsing the menus in Red Eclipse 2.0.0, I noticed the user interface scale settings were 50%, 75%, 100%, 150% and 200%. I was curious why there wasn't 125% and 175% options. I tried using the 150% setting, but the change didn't seem that big compared to 100%.

[It turns out the scale slider percentage is defined by the *area*, rather than a single dimension.](https://github.com/redeclipse/base/blob/85d625114e6eb2651d37f60e8ef32a0c92355022/config/ui/settings.cfg#L311-L324) This means that if you set the user interface scale to 150% (in Red Eclipse terms), each dimension is actually scaled to 123% (which is `sqrt(1.5)`).

Unfortunately, no other UI system I know of uses this kind of measurement, so I think it'd be better to use the size of a single dimension to follow the established convention (which is used by Windows, among other things). We can then add additional scaling options like 125% and 175%.

**Comment by ghost** [comment]
Yes you must pass from making `sqrt(1.5)` on both the width and the height if I'm not making any mistake I think to be right with conventional standards.
